### ISSUE TITLE
Prośba o dodanie funkcji: Miesięczna agregacja danych

### ISSUE THREAD

**Issue by NiceLeader** [Czy-Jest-Eldorado/feature-requests#30]
Prośba o dodanie funkcji: Miesięczna agregacja danych

Cześć,

Mam propozycję usprawnienia obecnych wykresów. Czy można dodać opcję wyświetlania danych w ujęciu miesięcznym zamiast codziennym? Chodzi o zagregowanie wyników tak, aby pokazywały sumaryczne wartości dla każdego miesiąca (np. średnie wynagrodzenie, liczba nowych/wygasłych ofert, aktywne oferty według typu), co ułatwi analizę długoterminowych trendów.

Dzięki temu dane byłyby bardziej czytelne i pozwalałyby skupić się na ogólnych zmianach bez potrzeby interpretacji codziennych fluktuacji.

**Comment by jkobus** [comment]
Tak, mamy w planach zmiany na stronie statystyk. Wstępnie myślałem o zaimplementowaniu czegoś takiego:
https://apexcharts.com/javascript-chart-demos/line-charts/zoomable-timeseries/

Ta sama biblioteka jest używana tutaj:
https://olxdata.azurewebsites.net/wszystkie/mieszkania

Ale oczywiście jestem otwarty na wszystkie pomysły.

**Comment by NiceLeader** [comment]
Dobrze. Ja polecam D3.js.

Myślę, że taki pomysł ma sens, ponieważ w praktyce użytkownicy rzadko patrzą na wykresy w kontekście krótkoterminowym, np. ostatnich 60 dni. Zdecydowanie częściej interesuje ich porównanie z poprzednimi miesiącami, sezonami czy latami, aby móc lepiej dostrzec długoterminowe trendy i zmiany.

**Comment by jkobus** [comment]
Hej, wracam do tematu - opublikowalismy nową stronę statystyk: https://czyjesteldorado.pl/raport-rynku-pracy-it

Zakres danych jest teraz nieco większy.

**Comment by NiceLeader** [comment]
Cześć, dobrze to wygląda
